homeassistant/components/august/camera.py

```diff
@@ -117,3 +126,20 @@ def device_info(self):
             "sw_version": self._firmware_version,
             "model": self._model,
         }
+
```

Comment:
Why does the async_update version of this entity store the firmware and the model, it's only needed for device info ? Why can't you just inline it there?

Reply:
I have to get it from self._data.async_get_doorbell_detail but that can do I/O so I didn't use it.  I could expose a new methods on AugustData for fetching lock and doorbell that do not do an update and refactor if that is better?

Reply:
@balloob  done in https://github.com/home-assistant/home-assistant/pull/32249